### PR TITLE
Change namespace delete command

### PR DESCRIPTION
**Pull request by jLopezbarb**
Signed-off-by: Javier López Barba <javier@okteto.com>
Change delete namespace to namespace delete because of depreciation

### CODE REVIEW

**Review by pchico83 — APPROVED** [review]

**Review by rberrelleza — COMMENTED** [review]
Do we need to update the cli version?

**Comment by jLopezbarb** [comment]
This PR will be merged when okteto CLI 1.15.0 is released, before that we can not merge it

**Pull request merged**
